handlers/lobbies.js

```diff
@@ -111,19 +111,11 @@ Dota2.Dota2Client.prototype.createPracticeLobby = function(options, callback) {
 
         if (this.debug) util.log("Sending match CMsgPracticeLobbyCreate request");
         var lobby_details = Dota2._parseOptions(finalOptions, Dota2._lobbyOptions);
-        lobby_details.pass_key = password;
-        lobby_details.leagueid = finalOptions.leagueid || tournament_id;
         var payload = {
             "lobby_details": lobby_details,
-            "pass_key": password
+            "pass_key": finalOptions.pass_key
```

Comment:
About this, looking at the nethook dumps it seems like the sent `pass_key` is an empty string. The one that matters is `lobby_details.pass_key`. I guess we could leave it like this just in case.